pyanaconda/ui/tui/spokes/network.py

```diff
@@ -563,26 +576,8 @@ def indirect(self):
         return True
 
     def apply(self):
```

Comment:
Btw. I think that the problem is that the `apply` method is called after every processed user input. You could change the `input` method and call `apply` only `if key == Prompt.CONTINUE`.

Reply:
I'll update the PR to do so.

Reply:
Done